File: Software Development Module/Week5/Guess.java
/*
Guess.java
Billy Cussen
13/07/2020
*/

public class Guess {
    
    //Data Members
    private int guess;
    private int correctNumber;
    private String result;

    //Constructor
    public Guess(int guess){
        this.guess = guess;
        this.correctNumber = 0;
        this.result = new String();
    }

    //Compute
    public String computeCorrectNumber(){

        this.correctNumber = (int) (Math.random()*10 + 1);
        //System.out.println("DEBUG: Random number is: "+this.correctNumber);

        if(this.guess == this.correctNumber){
            return "Correct";
        } else if(this.guess > this.correctNumber){
            return "Larger";
        } else{
            return "Smaller";
        }
    }

    //Getter
    public int getCorrectNumber(){
        return this.correctNumber;
    }

}

File: Python Course/Software Development Module - in Python/Week5/Guess.py
"""
Guess.py
Billy Cussen
27/09/2020
"""

import random

class myGuess:
    def __init__(self, number):
        self.number = number
        self.randomNumber = random.randint(1,10)

    def computeAnswer(self):
        if self.number == self.randomNumber:
            return "Correct"
        elif self.number < self.randomNumber:
            return "Too Low, correct number was "+str(self.randomNumber)
        else:
            return "Too High, correct number was "+str(self.randomNumber)